### DISCUSSION TITLE
Move or Jump without Stitch comand

### DISCUSSION THREAD

**Discussion by NakedFZR** [inkstitch/inkstitch#1206] [Q&A]
Hi I'm trying to use the inkstitch software to create multiple box pattern sews for webbing straps. The straps are clamped to a metal frame with windows for the sewing patterns. I need to move around the webbing clamping hardware from one pattern to the next (I cannot go in a straight line). My issue is that the jump stitches put in a stitch at the end of each jump. Is there a way to program movement paths without any stitching?

E.g. I need to move in a Z without putting the neadle down to get into the pattern window, sew the pattern, trim the thread, move to the next pattern in a Z without putting the neadle down. etc. 

I can move in a Z using jump stitches, but it puts in a stitch at each corner. Can I prevent it from putting in a stitch between jump stitches. 

![image](https://user-images.githubusercontent.com/7836664/118057172-ad79af00-b3ce-11eb-926a-5b2f408d5e7e.png)

**Comment by tyrosinase** [comment]
I do this on my in-the-hoop patterns, when I'm working around a big stack of dragons (trust me, it makes sense in context). I create a color change (the only way my machine will trim/halt; if your machine/format handles the stop command use that), my machine positions itself to start that color, and then I punch the "skip and move to the next color" button.

Of course this assumes you can manually advance through stitches without it doing a needle-down. I have no idea how common that function is. (Very common, I hope, because someday I'd like to sell this pattern.)

**Comment by lexelby** [comment]
If you have a multineedle, leave one needle unthreaded and set its thread break detector to the highest setting.  Then assign it to the moving sections, and it'll just poke holes :)

In a bit more serious vein, there is a feature I added to Ink/Stitch ages ago that might help, although I'm not even sure it still works.  You can set a "hoop out" position.  Then every time you have a stop command, it will jump to the "hoop out" position before stopping.

You set the hoop out position with a command icon (the Add Command extension).  Then attach a stop command to the last inject that should sew before the jumps.

**Comment by tyrosinase** [comment]
![IMG_7229](https://github.com/user-attachments/assets/17cecb91-5e47-43d1-959b-1039958d839a)

I promise, the embroidery frame is in there... somewhere (or at least the stabilizer; I might have taken the hoops themselves off before taking the picture).

That's sewing the heads for what eventually became the smaller six here (plus another of each color; the heads got sewn 12- or 16-up, depending): 

![IMG_7248](https://github.com/user-attachments/assets/865f4035-41a1-444c-80b3-e93ea1d93b22)
